scaffolding-chef-infra/lib/linux/run.bash

```diff
@@ -24,7 +24,7 @@ fi
 
 chef_client_cmd()
 {
-  chef-client -z -l "${cfg_log_level}" -c {{pkg.svc_config_path}}/client-config.rb -j {{pkg.svc_config_path}}/attributes.json --once --no-fork --run-lock-timeout "${cfg_run_lock_timeout}" "${cfg_chef_license_cmd}"
+  chef-client -z -l "${cfg_log_level}" -c {{pkg.svc_config_path}}/client-config.rb -j {{pkg.svc_config_path}}/attributes.json --once --no-fork --run-lock-timeout "${cfg_run_lock_timeout}" $cfg_chef_license_cmd
```

Comment:
I'm curious why we choose to get rid of `"{}"` as this is done intentionally to avoid globbing and some other bash issues.

Reply:
This was causing an issue where chef would fail to run the policyfile because it would thing that the empty string was an attempt to override the run-list which isn't supported.